dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -875,4 +875,18 @@
     public static final String METRICS_PROTOCOL = "metrics.protocol";
 
 
+    /**
+     * Serizlization ContentTypeId
+     */
+    public static final byte HESSIAN2_SERIALIZATION_ID = 2;
+    public static final byte JAVA_SERIALIZATION_ID = 3;
+    public static final byte COMPACTED_JAVA_SERIALIZATION_ID = 4;
+    public static final byte FASTJSON_SERIALIZATION_ID = 6;
+    public static final byte NATIVE_JAVA_SERIALIZATION_ID = 7;
+    public static final byte KRYO_SERIALIZATION_ID = 8;
+    public static final byte FST_SERIALIZATION_ID = 9;
+    public static final byte PROTOSTUFF_SERIALIZATION_ID = 10;
+    public static final byte AVRO_SERIALIZATION_ID = 11;
+    public static final byte GSON_SERIALIZATION_ID = 16;
+
```

Comment:
Maybe you can define a sub-constant class. What do you think?

Reply:
+1 for a specific constants class

Reply:
I think this suggestion is good, I will modify it later.